db/model/subject.js

```diff
@@ -315,8 +315,14 @@ module.exports = function subject(seq, dataTypes) {
            * is published and the sampleStoreFeature is enabled
            */
           if (featureToggles.isFeatureEnabled(sampleStoreFeature)) {
+            // Prevent any changes to original inst object data
+            const instData = JSON.parse(JSON.stringify(inst.get()));
```

Comment:
can we have the same variable name for this in here and aspect as well.?

Reply:
I changed the name because here we are getting only datavalues (using get) but in aspects we have full instance. Let me know if you still think we should have same var name and I will change it.

Reply:
Oh ok. I think we should be doing inst.get in aspects too.

Reply:
Yeah, I think so too, updated that! Thx!